go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 )
 
-//replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go dev
+replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v0.10.0
```

Comment:
I think you want to use the dev branch here, to get the latest additions to jfrog-client-go.

Reply:
For some reason, every time I change it to 'dev' IntelliJ overrides it to this specific version.